integration-tests/helpers/wallet-helpers.ts

```diff
@@ -23,25 +26,40 @@ import {
   createCredentialProvider,
 } from '@docknetwork/wallet-sdk-core/src/credential-provider';
 import {dockService} from '@docknetwork/wallet-sdk-wasm/src/services/dock';
+import {createDataStore} from '@docknetwork/wallet-sdk-data-store-typeorm/src';
 
 let wallet: IWallet;
 let didProvider: IDIDProvider;
 let credentialProvider: ICredentialProvider;
 let messageProvider: IMessageProvider;
 
-export async function createNewWallet(): Promise<IWallet> {
-  wallet = await createWallet({
+export async function createNewWallet({
+  dontWaitForNetwork,
+  otherProps,
+}: {
+  dontWaitForNetwork?: boolean;
+  otherProps?: CreateWalletProps;
+} = {}): Promise<IWallet> {
+  const dataStore = await createDataStore({
     databasePath: ':memory:',
     dbType: 'sqlite',
     defaultNetwork: 'testnet',
+    ...(otherProps ? otherProps : {}),
+  });
+
+  wallet = await createWallet({
+    dataStore,
   });
 
   didProvider = createDIDProvider({wallet});
   credentialProvider = createCredentialProvider({wallet});
   messageProvider = createMessageProvider({wallet, didProvider}) as any;
 
   console.log('Waiting for network connection');
-  await wallet.waitForEvent(WalletEvents.networkConnected);
+
+  if (dontWaitForNetwork) {
```

Comment:
this seems backwards to me. The boolean is `dontWait` but then right away we wait.